src/Components/test/E2ETest/Infrastructure/BrowserFixture.cs

```diff
@@ -29,7 +29,7 @@ public BrowserFixture()
 
             // On Windows/Linux, we don't need to set opts.BinaryLocation
             // But for Travis Mac builds we do
-            var binaryLocation = Environment.GetEnvironmentVariable("TEST_CHROME_BINARY");
+            var binaryLocation = System.Environment.GetEnvironmentVariable("TEST_CHROME_BINARY");
```

Comment:
Is this change still needed?

Reply:
Same with all the other places that add `System.` before `Environment`.

Reply:
Nope, I'll get rid of it.